bbc/simorgh
# Use updated Pashto locale to introduce relative time support

Resolves https://github.com/bbc/simorgh/issues/3147

**Overall change:** Use the latest `@bbc/psammead-locales` dependency, which contains an updated Pashto locale that supports relative timestamps. 

**Code changes:**

- Update `@bbc/psammead-locales` entry in `package.json` to the latest version.
- Add Pashto locale to Moment Locales story. http://localhost.bbc.com:9001/?path=/story/moment-locales-editorial-view--pashto-ps

**Manual testing performed:**

1. Use https://currentmillis.com/ to generate timestamps for:
    - A few seconds since epoch.
    - One minute since epoch.
    - Several minutes since epoch.
    - One hour since epoch.
    - Several hours since epoch.
    - Greater than 10 hours since epoch.
    - Greater than a day since epoch.
2. Change `firstPublished` and `lastPublished` values in `/data/pashto/articles/cng0e8v85eko.json` to verify the generated timestamps are displayed correctly in the application: http://localhost.bbc.com:7080/pashto/articles/cng0e8v85eko.
3. Verify the Pashto locale is displaying correctly in Storybook: http://localhost.bbc.com:9001/?path=/story/moment-locales-editorial-view--pashto-ps.

**Manual testing results:**

**A few seconds since epoch**
Although the Pashto locale supports this, `psammead-timestamp-container` doesn't. See: https://github.com/bbc/psammead/blob/latest/packages/containers/psammead-timestamp-container/src/timestampUtilities.js#L13-L20. Until that changes, the output is "A minute ago".
<img width="120" alt="seconds-ago" src="https://user-images.githubusercontent.com/1430787/65447359-46406d00-de2e-11e9-9285-a2178ac8b1ba.png">

**One minute since epoch** ✓
<img width="120" alt="a-minute-ago" src="https://user-images.githubusercontent.com/1430787/65447374-4e98a800-de2e-11e9-8e58-3b007be5345b.png">

**Several minutes since epoch** ✓
<img width="112" alt="several-minutes-ago" src="https://user-images.githubusercontent.com/1430787/65447382-52c4c580-de2e-11e9-9ddc-e919c8aad020.png">

**One hour since epoch** ✓
<img width="109" alt="an-hour-ago" src="https://user-images.githubusercontent.com/1430787/65447387-56f0e300-de2e-11e9-98eb-d4f3d1498999.png">

**Several hours since epoch** ✓
<img width="118" alt="several-hours-ago" src="https://user-images.githubusercontent.com/1430787/65447392-5d7f5a80-de2e-11e9-9abd-45267864713b.png">

**Greater than 10 hours since epoch (Absolute)**
Is this normal behaviour? Needs review.
<img width="182" alt="over-10-hours-ago" src="https://user-images.githubusercontent.com/1430787/65447396-61ab7800-de2e-11e9-928e-1884a0458958.png">

**Greater than a day since epoch (Absolute)** ✓
<img width="190" alt="one-day-one-hour-ago" src="https://user-images.githubusercontent.com/1430787/65447401-66702c00-de2e-11e9-9bf7-2a8fcb1f68b8.png">

---
